core/services/postgres/q.go

```diff
@@ -160,11 +167,13 @@ func (q Q) ExecQNamed(query string, arg interface{}) (err error) {
 func (q Q) Select(dest interface{}, query string, args ...interface{}) error {
 	ctx, cancel := q.Context()
 	defer cancel()
+	q.logSql(query, args...)
```

Comment:
It would also be nice to have these log something if the query takes longer than X seconds to execute - see commit 92268dc11dcbc922bdb695ece82b93da3a8395bd and file gorm_wrapper.go for examples

Reply:
Addressed this, although I will probably revisit this implementation to be more elegant.